CHANGELOG.md

```diff
@@ -72,6 +72,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed the Save/save as file type shows BIBTEX_DB instead of "Bibtex library" [#9372](https://github.com/JabRef/jabref/issues/9372)
 - We fixed an issue regarding recording redundant prefixes in search history. [#9685](https://github.com/JabRef/jabref/issues/9685)
 - We fixed an issue where passing a URL containing a DOI led to a "No entry found" notification. [#9821](https://github.com/JabRef/jabref/issues/9821)
+- we fixed an issue is wa delete an Automatic Keyword Group in the UI[#9778](https://github.com/JabRef/jabref/issues/9778)
```

Comment:
May I ask for a bit more care here? I know that writing text is not the most important thing for an IT professional. Nevertheless, this text is written by one person, but read by thousands of persons. We want to make the impression that JabRef is a careful engineered software. If we have typos and inconsistencies in that much-read file, it might hinder such an impression.

Moreover, time is limited. We need to take care about many things here in JabRef and want to focus on important things.

Compare with the line above. Here the correction:

```suggestion
- We fixed an issue is wa delete an Automatic Keyword Group in the UI. [#9778](https://github.com/JabRef/jabref/issues/9778)
```

Reply:
Oh, there was even more. This is outdated. See the comments of @Siedlerchr

Reply:
ok